### PR TITLE
docs: add list of topics for workshop

### PR DESCRIPTION
**Pull request by marrip**
here is my initial idea for the workshop - please give me some feedback 🥺

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
 
 presentation and resources for NorPreM bioinformatics workshop in March 2024
+
+## Project collaboration within NorPreM
+
+- setup
+  - github account
+  - addition to InPreD organisation
+- development
+  - short git introduction (basic git commands)
```
Comment (marrip) [inline README.md]: optional

File: README.md [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+  - devcontainers
```
Comment (marrip) [inline README.md]: skip

File: README.md [hunk 3]
```diff
@@ -1,3 +1,35 @@
...
+  - branching system (gitflow workflow)
+  - github actions (linting, testing, building)
+  - deployment (container image)
```
Comment (marrip) [inline README.md]: skip

File: README.md [hunk 4]
```diff
@@ -1,3 +1,35 @@
...
+  - pull requests (best practice)
+  - release and semantic versioning
+- python
```
Comment (marrip) [inline README.md]: after tso500_nxf_workflow

File: README.md [hunk 5]
```diff
@@ -1,3 +1,35 @@
...
+  - github account
+  - addition to InPreD organisation
+- development
+  - short git introduction (basic git commands)
+  - devcontainers
+  - branching system (gitflow workflow)
+  - github actions (linting, testing, building)
+  - deployment (container image)
+  - pull requests (best practice)
+  - release and semantic versioning
```
Comment (tinavisnovska) [inline README.md]: maybe we could add licensing as an additional point after the versioning

File: README.md [hunk 6]
```diff
@@ -1,3 +1,35 @@
 # 24-03_bioinfo_ws
 
 presentation and resources for NorPreM bioinformatics workshop in March 2024
```
Comment (tinavisnovska) [inline README.md]: I'd go with the full workshop name which is 'A two-day workshop for bioinformaticians and molecular biologists
with focus on the TSO500 pipeline in InPreD'

**Comment by marrip** [comment]
I had an idea - will use marp to generate the presentation so the `README.md` in this PR will be the single source of truth for the workshop presentation. The presentation will be available [here](https://inpred.github.io/24-03_bioinfo_ws/). To check layout and so on you can check there, for content you can check there or here in the actual markdown. Comments can be left in the markdown. I think that makes things much easier for us ☺️

**Comment by tinavisnovska** [comment]
Two topics that are not covered and we will have to take care of (I believe but am not 100% sure) when doing some PR are `git config user.name`, `git config user.email` (ideally with anonymised addressed) and github tokens.

**Comment by marrip** [comment]
> Two topics that are not covered and we will have to take care of (I believe but am not 100% sure) when doing some PR are `git config user.name`, `git config user.email` (ideally with anonymised addressed) and github tokens.

good point, will include this

**Pull request merged**